src/test/java/com/microsoft/graph/serializer/DefaultSerializerTests.java

```diff
@@ -99,4 +101,24 @@ public void testDeserializeDerivedType() throws Exception {
 		assertNotNull(o);
 		assertEquals("#microsoft.graph.fileAttachment", o. get("@odata.type").getAsString());
 	}
+  
+    @Test
+    public void testSerializerCanSerializeVoid() {
+        final DefaultSerializer serializer = new DefaultSerializer(new DefaultLogger());
+        Test1 t = new Test1();
+        String json = serializer.serializeObject(t);
+        // this line will emit a warning from Java 9 about illegal access to the constructor of Void 
+        // if gson TypeAdapterFactory is not handling Void properly
+        serializer.deserializeObject(json, Test1.class);
+    }
+  
+  public static final class Test1 {
```

Comment:
Can you give more meaningful name instead of Test1? how about TestClassWithVoidMember

Reply:
Ok, I'll call it `HasVoidMember`